packages/keystone/fields/SymmetricEncryptedText/Implementation.js

```diff
@@ -0,0 +1,64 @@
+const { Text } = require('@keystonejs/fields')
+
+const { CipherManager } = require('@open-condo/keystone/cipher')
+const { saveDecryptInfo, getDecryptInfo } = require('@open-condo/keystone/fields/SymmetricEncryptedText/utils/decryptInfo')
+
+class SymmetricEncryptedTextImplementation extends Text.implementation {
+
+    /** @type {CipherManager} */
+    cipherManager
+
+    constructor (path, options = {}) {
+        super(...arguments)
+
+        const errorStart = `${this.listKey}.${path}: SymmetricEncryptedText field`
+        
+        if (!options.cipherManager || !(options.cipherManager instanceof CipherManager)) {
+            throw new Error(`${errorStart} cipherManager with instance of CipherManager class required`)
```

Comment:
use GQLError

Reply:
Why? This intended to error on application start

Reply:
Ok